libraries/libfc/include/fc/string.hpp

```diff
@@ -2,151 +2,29 @@
 #include <fc/utility.hpp>
 #include <fc/fwd.hpp>
 
-#ifndef USE_FC_STRING
 #include <optional>
 #include <string>
+
 namespace fc
 {
-    typedef std::string string;
-
-  int64_t  to_int64( const fc::string& );
-  uint64_t to_uint64( const fc::string& );
-  double   to_double( const fc::string& );
-  fc::string to_string( double );
-  fc::string to_string( uint64_t );
-  fc::string to_string( int64_t );
-  fc::string to_string( uint16_t );
+  int64_t  to_int64( const std::string& );
+  uint64_t to_uint64( const std::string& );
+  double   to_double( const std::string& );
+  std::string to_string( double );
+  std::string to_string( uint64_t );
+  std::string to_string( int64_t );
+  std::string to_string( uint16_t );
   std::string to_pretty_string( int64_t );
-  inline fc::string to_string( int32_t v ) { return to_string( int64_t(v) ); }
-  inline fc::string to_string( uint32_t v ){ return to_string( uint64_t(v) ); }
+  inline std::string to_string( int32_t v ) { return to_string( int64_t(v) ); }
```

Comment:
Why cast int32_t to int64_t? Can those be templtaed?

Reply:
It think they can just be removed. See above comment. I'll create a follow on issue for this.
https://github.com/AntelopeIO/leap/issues/1057